ci/github/phpunit/mysqli.xml

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="../../../vendor/phpunit/phpunit/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.0/phpunit.xsd"
```

Comment:
This one can be kept as @greg0ire was mentioning once in https://github.com/doctrine/static-website-generator/pull/12#discussion_r1096587657

Reply:
All right.